package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "grace-shopper-funky-monkey",
-  "version": "0.7.1",
+  "version": "0.8.1",
```

Good you remembered.  Actually, this is a minor release so should be this. 

```suggestion
  "version": "0.8.0",
```